python_src/source/simulator/simulator.py

```diff
@@ -1,12 +1,48 @@
 import lgsvl
+import os
 
 
 class Simulator:
-    def __init__(self):
-        # Connect to the simulator. Load the scene from http://localhost:8080/#/Maps
-        sim = lgsvl.Simulator(address="localhost", port=8181)
-        if sim.current_scene == "SingleLaneRoad":
-            sim.reset()
-        else:
-            sim.load("SingleLaneRoad", seed=650387)
-        self.sim = sim
\ No newline at end of file
+    def __init__(self, seconds=30, scene="SingleLaneRoad", error_message=None, load_scene=True):
+        if error_message is None:
+            error_message = 'test timed out after {}s.'.format(seconds)
+        self.seconds = seconds
+        self.error_message = error_message
+        self.scene = scene
+        self.load_scene = load_scene
+
+    def __enter__(self):
+        self.sim = lgsvl.Simulator(os.environ.get("SIMULATOR_HOST", "127.0.0.1"), 8181)
```

Comment:
`Simulator` class provides a shortcut to `lgsvl instance`. I created `Simulator` class to simplify processes such as initialization, start and stop because those processes have been used many times. And in the future, it would be easy to add functions to write test cases to check whether the `lgsvl instance` is created correctly or not.